docs/source/references/cli_reference.rst

```diff
@@ -137,7 +137,7 @@ Manage stakes and other staker-related operations.
 +----------------------+-------------------------------------------------------------------------------+
 |  ``merge``           | Merge two stakes into one                                                     |
 +----------------------+-------------------------------------------------------------------------------+
-|  ``remove-unused``   | Remove unused stake                                                           |
+|  ``remove-inactive`` | Remove inactive stake                                                         |
```

Comment:
We should also update the heading here: https://github.com/nucypher/nucypher/blob/main/docs/source/staking/stake_management.rst#remove-unused-sub-stake.

Reply:
✔️

Reply:
One more spot: https://github.com/nucypher/nucypher/blame/main/docs/source/staking/stake_management.rst#L224

Reply:
✔️ Thank you for keeping an eye out! I need to remember to disable the `.py` filter on my full text search.